numba_dppy/tests/integration/test_dpnp_interop.py

```diff
@@ -44,37 +44,27 @@ def usm_type(request):
     return request.param
 
 
-def test_dpnp_create_array_in_context(offload_device, dtype):
-    if not ensure_dpnp():
-        pytest.skip("No DPNP")
-
+@pytest.mark.parametrize("filter_str", filter_strings)
+def test_dpnp_create_array_in_context(filter_str, dtype):
     import dpnp
```

Comment:
Previously `ensure_dpnp():` checked that we can `import dpnp`. Now where is this check? If it is on module level then why we do not `import dpnp` on module level too?

Reply:
On module level tests skips if  **import dpnp_fptr_interface** doesn't works, at that test check all dpnp import

Reply:
Check `import dpnp` on module level. And import dpnp only one time at the top of the module.